playbooks/common/openshift-cluster/config.yml

```diff
@@ -26,6 +26,21 @@
   tags:
   - always
 
+- name: Setup firewall
+  hosts: oo_all_hosts
+  tags:
+  - always
+  tasks:
+  # This should move to intialize_facts
+  - name: set os_firewall_enabled
```

Comment:
Why are these fact being set here?  Is there an issue with how the defaults from the role are being applied?